environments/prod/hearings-hmcts-net.yml

```diff
@@ -352,12 +352,12 @@ cname:
 
 
 # prod
-  - name: _b23665c45f36445a89205a205c2c9fdd
+  - name: _BAC10B76E9914287641A796A4D75DEF8
```

Comment:
can you lowercase the name record? terraform will keep trying to change it otherwise

Reply:
can we change the terraform to use tolower. The config comes in this format and I think its a bit unnecessary to ask humans to correct this when we have code that can do this for us

Reply:
sure, could you make the change please?